doc/connectivity/bluetooth/api/mesh/brg_cfg.rst

```diff
@@ -0,0 +1,150 @@
+.. _bluetooth_mesh_brg_cfg:
+
+Subnet Bridge
+#############
+
+With Bluetooth Mesh Protocol Specification version 1.1, the Bluetooth Mesh Subnet Bridge feature was
+introduced.
+This feature allows mesh networks to use subnets for area isolation, but to also allow communication
+between specific devices in different, adjacent subnets without compromising security.
+
+The Bluetooth Mesh Subnet Bridge feature makes it possible for selected nodes in the network to act
+as Subnet Bridges, allowing controlled communication by relaying messages between nodes in adjacent
+subnets.
+
+The Subnet Bridge feature includes two models:
+
+* :ref:`bluetooth_mesh_models_brg_cfg_srv`
```

Comment:
I see we now have both * and - for bullet point in the same file.
I would consider aligning on one of them, not that it has any effect for the rendering (it should still be rendered the same) :)